### DISCUSSION TITLE
Updates with the release of a new version of the client and patch

### DISCUSSION THREAD

**Discussion by Nimmo89** [NebulaModTeam/nebula#611] [General]
Good evening, this project is still working and is it planned to update it to the current version of the client, which is scheduled for December 15, 2023? It’s just that, as I understand it, no one decided to take on such a grandiose development except you.

**Comment by starfi5h** [comment]
Yes, it is still developing. It may split into a few stages to sync with different systems. The first goal is to make the 0.10.x game without Dark Fog works and has all non-battle settings sync.

**Comment by kenshisaan** [comment]
thanks !

**Comment by shawjiadi** [comment]
Could you please let me know when we can expect the release of content that is compatible with the current version?

**Comment by kenshisaan** [comment]
They are working hard, follow there news on their discord.
there no release window

**Reply by shawjiadi** [reply]
thanks !
